lib/monitoring/aws-rds/AuroraClusterMonitoring.ts

```diff
@@ -0,0 +1,239 @@
+      const createdAlarm =
+        this.connectionAlarmFactory.addMaxConnectionCountAlarm(
+          this.connectionsMetric,
+          alarmProps,
+          disambiguator
+        );
+      this.connectionAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+
+    for (const disambiguator in props.addMaxServerlessDatabaseCapacityAlarm) {
+      const alarmProps =
+        props.addMaxServerlessDatabaseCapacityAlarm[disambiguator];
+      const createdAlarm =
+        this.auroraAlarmFactory.addMaxServerlessDatabaseCapacity(
+          this.serverlessDatabaseCapacityMetric,
+          alarmProps,
+          disambiguator
+        );
+      this.connectionAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+
+    props.useCreatedAlarms?.consume(this.createdAlarms());
+  }
+
+  summaryWidgets(): IWidget[] {
+    return [
+      this.createTitleWidget(),
+      this.createCpuUsageWidget(QuarterWidth, DefaultGraphWidgetHeight),
+      this.createConnectionsWidget(QuarterWidth, DefaultGraphWidgetHeight),
+      this.createLatencyWidget(QuarterWidth, DefaultGraphWidgetHeight),
+      this.createAuroraMetricsWidget(QuarterWidth, DefaultGraphWidgetHeight),
+    ];
+  }
+
+  widgets(): IWidget[] {
+    return [
+      this.createTitleWidget(),
+      this.createCpuUsageWidget(QuarterWidth, DefaultGraphWidgetHeight),
+      this.createConnectionsWidget(QuarterWidth, DefaultGraphWidgetHeight),
+      this.createLatencyWidget(HalfWidth, DefaultGraphWidgetHeight),
+      this.createAuroraMetricsWidget(HalfWidth, DefaultGraphWidgetHeight),
+    ];
+  }
+
+  createTitleWidget() {
+    return new MonitoringHeaderWidget({
+      family: "Aurora Cluster",
+      title: this.title,
+      goToLinkUrl: this.url,
+    });
+  }
+
+  createCpuUsageWidget(width: number, height: number) {
+    return new GraphWidget({
+      width,
+      height,
+      title: "CPU Usage",
+      left: [this.cpuUsageMetric],
+      leftYAxis: PercentageAxisFromZeroToHundred,
+      leftAnnotations: this.usageAnnotations,
+    });
+  }
+
+  createConnectionsWidget(width: number, height: number) {
+    return new GraphWidget({
+      width,
+      height,
+      title: "Connections",
+      left: [this.connectionsMetric],
+      leftYAxis: CountAxisFromZero,
+      leftAnnotations: this.connectionAnnotations,
+    });
+  }
+
+  createLatencyWidget(width: number, height: number) {
+    return new GraphWidget({
+      width,
+      height,
+      title: "Query Duration",
+      left: [
+        this.selectLatencyMetric,
+        this.insertLatencyMetric,
+        this.updateLatencyMetric,
+        this.deleteLatencyMetric,
+        this.commitLatencyMetric,
+      ],
+      leftYAxis: TimeAxisMillisFromZero,
+    });
+  }
+
+  createAuroraMetricsWidget(width: number, height: number) {
+    return new GraphWidget({
+      width,
+      height,
+      title: "Aurora Specifc Cluster Metrics",
```

Comment:
Feels like a bit of an implementation detail. Maybe just specify "Serverless Database Capacity"? We can find a better name if anything's ever added later on.